lte/protos/session_manager.proto

```diff
@@ -919,25 +949,24 @@ message M5GSMSessionContextAccess{
     // TS-24501 - 9.11.4.3 - minimum 7 and max 65538, variable length
     repeated QosRules authorized_qos_rules = 4;
     // TS-24501 - 9.11.4.14 PDU session  Bit Rate imported from subscriberdb.proto
-    AmbrUnit downlink_unit_type = 5;
-    // Only to use lower 2 bytes (16 bit values)
-    uint32   downlink_units = 6;
-    AmbrUnit uplink_unit_type = 7;
-    // Only to use lower 2 bytes (16 bit values)
-    uint32   uplink_units = 8; 
-    // TS-24501 - 9.11.4.2
-    M5GSMCause M5gsm_cause = 9;
-    // PDU session is requested to be established as always-on PDU session.
-    bool always_on_pdu_session_indication = 10;
-    // Indicates SSC mode.
-    SscMode allowed_ssc_mode = 11;
-    bool M5gsm_congetion_re_attempt_indicator = 12;
-    // IP address. Imported from session_manager.proto file.
-    RedirectServer pdu_address = 13;
-    TeidSet upf_endpoint = 14;
-    // Procedure transaction identity
-    bytes procedure_trans_identity = 15;
-    FlowQos  qos =16;
+
+    //TS-24501 - 9.11.4.14 PDU session Aggregate Maximum Bit Rate imported from subscriberdb.proto
```

Comment:
Can you put comment above the message so that it gets included into the generated files as well?

Ex:
```
M5GSMCause M5gsm_cause = 10; //TS-24501 - 9.11.4.2
```
to
```
// TS-24501 - 9.11.4.2
M5GSMCause M5gsm_cause = 10;
```

Reply:
And also let's do a consistent single space between `//` and the comment

Reply:
resolved